aten/src/ATen/native/Blas.cpp

```diff
@@ -137,6 +140,110 @@ Tensor dot(const Tensor &self, const Tensor &other){
   });
 }
 
+// this is to port LDA_COND from the TH codebase
```

Comment:
nit: 

```
// NOTE: this is a port of LDA_COND from the TH codebase...
```

Reply:
Thanks for pointing this out, I will address this in the later commit.